test/unittest/dds/profiles/test_xml_profile.xml

```diff
@@ -349,7 +356,6 @@
                 </partition>
                 <publishMode>
                     <kind>ASYNCHRONOUS</kind>
-                    <flow_controller_name>test_flow_controller</flow_controller_name>
```

Comment:
This was ok, I would keep it.

Reply:
In this case, as it is not the default profile, we should keep the default `flow controller name`. Otherwise some Publisher tests will fail as the participant profile is not loaded and does not register the flow_controllers

Reply:
Fair enough!